### PR TITLE
chore: Automate Releases

### PR DESCRIPTION
**Pull request by duncanista**
# What?

Automate sandbox and production releases in GitLab.

# Motivation

Save hundreds of engineering hours and manual work.

# How?

Generate pipelines by leveraging templates. a
Automated production release process will be explained in internal documentation.

### CODE REVIEW

**Review by alexgallotta — APPROVED** [review]
Looks great! A few improvements could be done later on, like clean up and reuse code in bash, making it more concise, readable and maintainable

File: .gitlab/datasources/regions.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - code: "eu-north-1"
+  - code: "eu-west-1"
+  - code: "eu-west-2"
+  - code: "eu-west-3"
+  - code: "eu-south-1"
+  - code: "eu-south-2"
+  - code: "il-central-1"
+  - code: "me-south-1"
+  - code: "me-central-1"
+  - code: "sa-east-1"
```
Comment (astuyve) [inline .gitlab/datasources/regions.yaml:30]: did we double check this list is correct? We had a minor issue where we missed a couple regions earlier.

**Review by astuyve — APPROVED** [review]
Okay, let's ship it.

File: .gitlab/datasources/regions.yaml (hunk 1 above)
Comment (duncanista) [reply .gitlab/datasources/regions.yaml:30]: Yup, I manually verified this before adding this file!

**Pull request merged**